spec/order_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'order'
+
+RSpec.describe Order do
+  let(:order) { described_class.new(menu) }
+  let(:menu) { double :menu, :dishes => dishes_on_menu }
+  let(:dishes_on_menu) do {
+      spring_rolls: 5,
+      wonton: 10,
+      congee: 10,
+      chow_mein: 15,
+      hotpot: 20
+    }
```

Comment:
You could combine this: 
@dishes_on_menu = {spring_rolls: 5, wonton: 10, congee: 10, chow_mein: 15, hotpot: 20}
@menu = double("Menu", dishes: {@dishes_on_menu)... I don't think you need to use 'let' syntax

Reply:
Also in your code you only ever order congee and wontons so you can actually have these as the only dishes on the menu without compromising your code. Doubles don't need to actually look like a real instance of that class, they just need to be the simplest form possible to work with the class your testing